README.md

```diff
@@ -19,74 +19,94 @@ These are related projects that are either already setup and available for use o
+2. Find your VM and make it active by clicking anywhere in that line.
+3. Under the drop-down *Actions*, go to *Networking*, *Manage IP Addreses*.
+4. Click on *Allocate an Elastic IP*. It will automatically create an IP address and show it on the screen.
+5. Write down that _Elastic IP_. At this point, whatever IP address is shown under IPv4 Public IP for your VM is not the current instance's IP address. Next you need to associate the _Elastic IP_ with your VM.
+6. Back in _EC2 Dashboard_ go to *Elastic IPs*. The IP address just created should be in that list. Check it and under *Actions*, *Associate*, choose resource type "Instance", and choose your EC2 (e.g., searching by its name).
+7.  <a name="sshconnect"></a>In _EC2 Dashboard_ make your VM active by clicking it. Then click _Connect_ on top. The example in that window shows you how to ssh into your VM from a terminal.
 
-* assign an Elastic IP (a static IP address) to your instance
-* open inbound ports on your security group
-    * 80 <- world
-    * 22 <- world
-    * 443 <- world
-    * all TCP <- the elastic IP of the VM (Make sure you add /32 to the Elastic IP)
-    * all TCP <- the security group itself
 
-#### Adding private SSH key to your VM
+#### Adding a private SSH key to your VM
 
-Add your private ssh key under `~/.ssh/<your_key>.pem`, this is typically the same key that you use to SSH to your host VM, regardless it needs to be a key created on the AWS console so Amazon is aware of it. Then do `chmod 400 ~/.ssh/<your_key>.pem` so your key is not publicly viewable.
+Add your private ssh key under `~/.ssh/<your_key>.pem`, this is typically the same key that you use to SSH to your host VM, regardless it needs to be a key created on the AWS console so Amazon is aware of it. Then set privileges to _read-by-user-only_ by `chmod 400 ~/.ssh/<your_key>.pem` so your key is not publicly viewable.
 
-#### TODO:
 
-* Guide on choosing AWS instance type... make sure it matches your AMI.
-* AMI, use an ubuntu 16.04 base box, you can use the official Ubuntu release.  You may need to make your own AMI with more storage! Needs to be in your region!  You may want to google to start with the official Ubuntu images for your region.
+## Installing the Platform (CGP)
 
-### Setup for Boardwalk
+These directions below assume you are using AWS.  We will include additional cloud instructions as `cgp-deployment` matures.
 
-See the Boardwalk [README](boardwalk/README.md) for details.
+
+### Collecting Information
+
+The installation script (`install_bootstrap`) will prompt for several questions. To expedite the installation process it is useful to prepare for some of the answers beforehand. Here are a few pointers:
+
+* make sure you know what AWS region your VM runs in (e.g. `us-west-2`)
+* decide whether you want to create an instance for development or production as it might impact the size and therefore the cost of the host virtual machine (see above for recommendations)
```

Comment:
I think you cover this above, and don't need this here.

It seems out of place, because you are talking about collecting information to answer the `install_bootstrap` questions, but at this point, you've already set up the VM.

Reply:
Not sure (but this could be misinterpreting again your meaning/intention). But this was my original motivation for re-writing the readme. I've always kept notes on how to answer the installer questions. This laundry list serves the purpose of keeping such a list handy for the user to make the installation smoother. You assume that the order always is (i) setup a vm and (ii) run the installer, when in reality is often is (i) set up the vm, and (ii ... n) run the installer, and when you run the installer for the k-th time you forgot that your vm runs in us-west-2... Apologies for the long-winded answer. I leave it in for now until I hear your response.